content/for-teams.de.html

```diff
@@ -2,4 +2,5 @@
 title: "Cryptomator Hub"
 type: for-teams
 navtitle: "Für Teams"
+aliases: ["de/hub"]
```

Comment:
```suggestion
aliases: ["/de/hub"]
```

Not sure if this is necessary, but let's keep it consistent.

Reply:
Fixed in 278a2ff.